api/structs/release.go

```diff
@@ -19,3 +19,11 @@ func NewRelease(app string) *Release {
 		Id:  generateId("R", 10),
 	}
 }
+
+func (rs Releases) Latest() *Release {
```

Comment:
This could be susceptible to bugs in the future. There's no guarantee that this `Releases` struct is sorted appropriately.